spacebench/algorithms/spatialplus.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from statsmodels.gam.api import GLMGam, BSplines
+import pandas as pd
+import statsmodels.api as sm
+
+def spatial_plus(
+        X: np.ndarray,
+        Y: np.ndarray,
+        coord: np.ndarray,
+        df: pd.DataFrame
+        ):
+    """Spatial plus algorithm.
+
+    Arguments
+    ---------
+    X: np.ndarray 
+        A vector of exposures (could be binary or continuous).
+    Y: np.ndarray
+        A matrix of outcomes.
```

Comment:
How can the outcomes be a matrix? Aren't they a vector?

Reply:
Whoops, should be a vector, (1d numpy array) accidentally followed copilot's autocomplete

Reply:
Happens to me a lot lol.